frontend/javascripts/dashboard/dataset/dataset_rotation_form_item.tsx

```diff
@@ -80,9 +87,14 @@ export const AxisRotationFormItem: React.FC<AxisRotationFormItemProps> = ({
       if (datasetBoundingBox == null) {
         return;
       }
-
-      const rotationInRadians = rotationInDegrees * (Math.PI / 180);
-      const rotationMatrix = getRotationMatrixAroundAxis(axis, rotationInRadians);
+      const rotationValues: RotationSetting = form.getFieldValue(["datasetRotation"])[axis];
+      if (rotationInDegrees !== undefined) {
+        rotationValues.rotationInDegrees = rotationInDegrees;
+      }
+      if (isMirrored !== undefined) {
+        rotationValues.isMirrored = isMirrored;
```

Comment:
this directly mutates the return value from `form.getFieldValue`. is this how it should be done with antd forms?